HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -163,6 +163,14 @@ def customiseFor18429(process):
 
      return process
 
+# Add new parameters to RecoTrackRefSelector
+def customiseFor19029(process):
+    for producer in producers_by_type(process, "RecoTrackRefSelector"):
+        if not hasattr(producer, "minPhi"):
+            producer.minPhi = cms.double(-3.2)
+            producer.maxPhi = cms.double(3.2)
```

Comment:
I know... But as far as I can tell, the object selector templates do not offer fillDescriptions() at the moment (and I don't have time now to try that).

https://github.com/cms-sw/cmssw/blob/master/CommonTools/RecoAlgos/plugins/RecoTrackRefSelector.cc#L19
https://github.com/cms-sw/cmssw/blob/master/CommonTools/UtilAlgos/interface/ObjectSelectorStreamProducer.h#L13
https://github.com/cms-sw/cmssw/blob/master/CommonTools/UtilAlgos/interface/ObjectSelectorProducer.h
https://github.com/cms-sw/cmssw/blob/master/CommonTools/UtilAlgos/interface/StoreManagerTrait.h#L92-L103